contrib/masterfiles-stage/README.org

```diff
@@ -101,6 +105,15 @@ when developing new staging methods.
     - 88335d36b48c8808b12b48667a463182dc8d0338
     - mytag
 
+- channel_config_file - The path to the config file for multiple policy channels
+  - Example value:
+    - /var/cfengine/policy_channels/channel_to_source.txt
```

Comment:
Could this be done with bash arrays in the params or options instead of a new separate file?